planner/core/logical_plans_test.go

```diff
@@ -173,8 +173,8 @@ func (s *testUnitTestSuit) TestIndexPathSplitCorColCond(c *C) {
 			corColIDs:  []int64{},
 			idxColIDs:  []int64{3},
 			idxColLens: []int{types.UnspecifiedLength},
-			access:     "[]",
-			remained:   "[eq(Column#3, \x01)]",
+			access:     "[eq(Column#3, \x01)]",
```

Comment:
@winoros @xuyifangreeneyes PTAL, I am sure the test is correct.

Reply:
If the constant side should evaled by utf8mb4 then it's correct.